src/components/scenes/TransactionListScene.tsx

```diff
@@ -59,6 +62,7 @@ function TransactionListComponent(props: Props) {
   const tokenId = checkToken(route.params.tokenId, wallet.currencyConfig.allTokens)
   const { pluginId } = wallet.currencyInfo
   const { currencyCode } = tokenId == null ? wallet.currencyInfo : wallet.currencyConfig.allTokens[tokenId]
+  const { displayName } = tokenId == null ? wallet.currencyInfo : wallet.currencyConfig.allTokens[tokenId]
```

Comment:
You could just delete this line, and do `const { currencyCode, displayName } = ...` above.